RedHat-UX/red-hat-design-system
# docs: bundle rhds and pfe

## What I did

1. bundle PFE with RHDS for docs builds, which allowed me to
2. remove the `pfe.min.js` module from docs pages
3. remove some pfe 11ty plugins and substitute our own
4. put docs page shortcodes (example, section, demo) in their own plugin file

Fixes #300 

## Testing Instructions

1. start 11ty in watch mode
    ```bash
    npm run clean
    npm ci
    npm run watch:docs
    ```
2. accordion and other custom elements should load
